runtime/expr/agg/ztests/math-coercion.yaml

```diff
@@ -0,0 +1,13 @@
+zed: 'sum(this), max(this), min(this), avg(this)'
+
+input: '2 2.5'
+
+output-flags: '-pretty 4'
```

Comment:
```suggestion
zed: sum(this), max(this), min(this), avg(this)

input: |
  2
  2.5

output-flags: -pretty=4
```

Reply:
I get getting rid of the quotes on zed and output-flags but what does breaking the input into multiple lines give us?

Reply:
Makes it clearer that there are two inputs, but no offense taken if you prefer it as is.